atomate/vasp/workflows/presets/core.py

```diff
@@ -346,3 +349,114 @@ def wf_thermal_expansion(structure, c=None):
         wf = add_wf_metadata(wf, structure)
 
     return wf
+
+
+def wf_nudged_elastic_band(structures, parent, c=None):
```

Comment:
see my previous comment about "parent" and about how, in my understanding, a fully automatic NEB would only require two endpoint structures.